src/main/default/lwc/lookup/lookup.js

```diff
@@ -1,16 +1,19 @@
 import { LightningElement, api } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
 
 const MINIMAL_SEARCH_TERM_LENGTH = 2; // Min number of chars required to search
 const SEARCH_DELAY = 300; // Wait 300 ms after user stops typing then, peform search
 
-export default class Lookup extends LightningElement {
+export default class Lookup extends NavigationMixin(LightningElement) {
     @api label;
     @api required;
     @api placeholder = '';
     @api isMultiEntry = false;
     @api errors = [];
     @api scrollAfterNItems;
     @api customKey;
+    @api newSobjectName;
+    @api isDisabled;
```

Comment:
Let's be consistent with standard attribute names: please renamed that to `disabled`.